apps/web/package.json

```diff
@@ -167,6 +167,7 @@
     "msw": "^0.42.3",
     "postcss": "^8.4.18",
     "tailwindcss": "^3.3.1",
+    "tailwindcss-animate": "^1.0.6",
```

Comment:
Wondering if we measured the impact of adding such plugins...

Reply:
Its just a CSS addition - will just increase css bundle if anything. Very miniscule

Reply:
And since its tailwind - JIT just compiles the classes down to only the ones we use. This might add 2 css classes